person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
 
   def initialize(*args)
     @id = Random.rand(1..1000)
     @age = args[0]
     @name = args[1] || 'unknown'
     @parent_permission = args[2].nil? ? true : args[2]
+    @rentals = []
     super()
```

Comment:
- [ ] You should add a method for adding rentals. Here the person itself should be sent as a parameter to create the new rental achieving the `has-many` association.